app/carbon-fields/cf-theme-options.php

```diff
@@ -111,6 +104,15 @@ function crb_attach_theme_options()
                     'compare' => '='
                 ]])
                 ->set_width(33),
+            Field
+                ::make('html', 'cached_version')
+                ->set_html(__('<h3 style="font-size:13px; margin-top: 0;">Cached LOCAL styles version</h3><p style="font-size:13px">'.\App\Core\ALPSVersions::getLocalCachedVersion().'</p>', 'alps'))
```

Comment:
Change the heading the say `Cached LOCAL style version` and the next line to say `ALPS Version n.n.n`

Reply:
Thank you. It was fixed